index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio Website</title>
+</head>
+<body>
+    <div id="wrapper">
+        <header id="container">
+            <div>
+                <p><a class="logo" href="#">Suleiman</a></p>
+            </div>
+            <div id="navbar">
+                <a href="#"><img src="images/Hamburguer.png" alt="an image here" class="image"></a>
+            </div>
+        </header>
```

Comment:
- [ ] [**OPTIONAL**] Well done @shaaibu7 , Always make sure your design stick to the Figma design. I know how much effort you put in that, however, there is one thing previous code reviewer forget to  tell you, and I'm sorry about that.  You should adjust your ```navbar``` so that it becomes centred as  in the Figma design :+1: 
Your Design

![Screenshot from 2022-12-02 07-45-28](https://user-images.githubusercontent.com/80458199/205224943-c40d0be9-f1b2-4be8-9186-db3e852ab238.png)

The Figma design    

![Screenshot from 2022-12-02 07-43-54](https://user-images.githubusercontent.com/80458199/205224834-83636597-f340-46a8-8e0e-77f102f76cde.png)